src/lib/agoric.ts

```diff
@@ -230,13 +230,12 @@ export const queryParams = async () => {
     );
 
     try {
-        let capDataStr = await vstorage.readLatest("published.fastUsdc.feedPolicy")
-        const { value } = JSON.parse(capDataStr);
-        const specimen = JSON.parse(value);
+        let capDataObj = await vstorage.readLatest("published.fastUsdc.feedPolicy")
+        const specimen = JSON.parse(capDataObj.value);
         const { values } = specimen;
```

Comment:
For the short term, I think we want this code to accommodate `readLatest` outputting either a string or an object.
```suggestion
        let wrapper = await vstorage.readLatest("published.fastUsdc.feedPolicy");
        if (typeof wrapper === "string") wrapper = JSON.parse(wrapper);
        const streamCell = JSON.parse(wrapper.value);
        mustMatch(streamCell, StreamCellShape);
        const { values } = streamCell;
```
(and likewise for reading "published.fastUsdc")